code/game/objects/items/weapons/implants/implantcase.dm

```diff
@@ -54,45 +55,6 @@
 				update_state()
 			I.update_icon(UPDATE_ICON_STATE)
 
-	/*else if(istype(W, /obj/item/ammo_casing/shotgun/implanter))
-		var/obj/item/ammo_casing/shotgun/implanter/I = W
-		if(I.implanter)
-			src.attackby(I.implanter, user, params) */ // COMING SOON -- c0
```

Comment:
coming soon...

this would have been pretty damn cool, if not incredibly overpowered. Mindshield implant slugs.

Reply:
adressed